Documented feature for Restrict a custom repository to RHEL 9 in Cont… (#1026)
theforeman/foreman-documentation: guides/common/modules/proc_restrict-custom-repository-RHEL9.adoc

[id="Restricting_Custom_Repository_to_RHEL9{context}"]
= Restricting Custom Repository Sets to RHEL 9 in {Project}

.Procedure

> . In the {ProjectWebUI}, navigate to *Content* > *Products* and click the product that you want to restrict.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Products* and click the product that contains the repository sets that you want to restrict.
```

. In the *Repositories* tab, click the repository that you want to restrict.

> . Navigate to *Sync Settings* > *Restrict to OS*  and select the operating system from the list to include RHEL operating systems for their corresponding RHEL repositories and click *Save*.

Reviewer: ```suggestion
. Navigate to *Sync Settings* > *Restrict to OS*  and select the operating system from the list to include {RHEL} operating systems for their corresponding {RHEL} repositories and click *Save*.
```

[id="Restricting_Custom_Repository_to_RHEL9{context}"]
= Restricting Custom Repository Sets to RHEL 9 in {Project}

> Within a Product where a user is preparing to add repositories to that product, the user can restrict the repositories by operating system. Under Sync Settings, use the Restrict to OS version list to include RHEL operating systems for their corresponding RHEL repositories.

Reviewer: You can delete all of this really.

.Procedure

> . In the {ProjectWebUI}, navigate to *Content* > *Products* and click on the desired product under *Name*.

Reviewer: We never ever use desire. 
```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Products* and click the product that you want to restrict.
```
There's no desire in tech docs. 
IBM Style has the following entry for desire: 

desire v
Do not use. Use “want.”

pp 321

We also never use "under" "over" etc. IBM Style describes exactly how to approach these scenarios.

> . In the *Repositories* tab, click on the desired repository under *Name*.

Reviewer: ```suggestion
. In the *Repositories* tab, click the repository that you want to restrict. 
```
Same issues as above.
